alpha/action/diff_test.go

```diff
@@ -311,6 +311,77 @@ packages:
 `,
 			assertion: require.Error,
 		},
+		{
```

Comment:
Maybe it's worth adding a happy path test too?

Reply:
Unfortunately, `semver.Range` is a func so asserting equality of two ranges as objects in this test is just not possible. I prefer not to change this test just for that. Instead, on the other `diff_test`, the test cases will validate that correct range will behave as expected.